gammapy/utils/interpolation.py

```diff
@@ -239,7 +239,7 @@ def _inverse(cls, values):
         return values
 
 
-def interpolate_profile(x, y, interp_scale="sqrt"):
+def interpolate_profile(x, y, interp_scale="sqrt", extrapolate=False):
```

Comment:
Can you add in the docstring the type of x and y, as well as the return value/quantity?

Reply:
Added a docstring entry for extrapolate.
For x, y the existing docstring already mentions they are arrays but we cannot say anything about the unit as it is a generic class.